src/Exception/ExceptionConverter.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\FtpExtractor\Exception;
+
+use Keboola\Component\UserException;
+use League\Flysystem\FileNotFoundException;
+
+final class ExceptionConverter
+{
+    public static function handleCopyFilesException(\Throwable $e): void
+    {
+        // phpcs:disable
+        if (preg_match_all('/(Could not login)|(getaddrinfo failed)|(Could not connect to)|(Cannot connect to)|(Root is invalid)|(The authenticity of)/', $e->getMessage())) {
```

Comment:
Tyhle už nejsou SftpAdapterException?

Reply:
Jsou https://github.com/thephpleague/flysystem-sftp/search?q=Could+not+login&unscoped_q=Could+not+login ale taky se ukázalo že jsou i v  `thephpleague/flysystem` https://github.com/thephpleague/flysystem/search?q=Could+not+login&unscoped_q=Could+not+login

Reply:
Jo, jasně. Pošleš PR i tam? Tady je to teda takhle OK.

Reply:
Pošlu i tam, zkusím ještě dnes :)